dotnet-desktop-guide/net/winforms/printing/how-to-print-windows-form.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "How to print a Windows Form"
+description: Learn how to programmatically print a copy of the current Windows Form by using the CopyFromScreen method.
+ms.date: "05/04/2022"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "Windows Forms, printing"
+  - "printing [Windows Forms]"
+  - "printing a form"
+  - "printing [Windows Forms], printing a form"
+ms.custom: devdivchpfy22
+---
+
+# How to print a Windows Form
```

Comment:
This should use the correct syntax for this repo. Considering "Windows Form" will be repeated, I would just simplify it to `How to print a Form`.
```suggestion
# How to print a Form (Windows Forms .NET)
```